images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -205,7 +211,7 @@ function Get-DotnetTools {
     $toolsList = @()
 
     foreach  ($dotnetTool in $dotnetTools) {
-        $toolsList += $dotnetTool.name + " " + (Invoke-Expression $dotnetTool.getversion)
+        $toolsList += [ToolVersionNode]::new($dotnetTool.name, (Invoke-Expression $dotnetTool.getversion))
```

Comment:
nit: could you please move `Invoke-Expression` to the separate line / temporary variable?

Reply:
ok, moved to temp var